rules/no-identical-alternate.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const {isSameReference} = require('./utils/index.js');
+
+const MESSAGE_ID = 'no-identical-alternate';
+const messages = {
+	[MESSAGE_ID]: 'Replace nested ternary expressions with a logical expression.',
```

Comment:
Should be more specific about what kind of ternary expression it targets.

Reply:
How about `Replace repeated alternatives of ternary expressions with a logical expression.` or `Simplify repeated alternatives of ternary expressions.` ?